doc/conf.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+
+from nbsite.shared_conf import *
+
+###################################################
+# edit things below as appropriate for your project
+
+project = u'EarthML'
+authors = u'Anaconda'
+copyright = u'2018 ' + authors
+description = 'Python tools for earth science and machine learning'
+
+version = '0.0.1'
+release = '0.0.1'
+
+nbbuild_cell_timeout = 10000
+
+html_static_path += ['_static']
+html_theme = 'sphinx_ioam_theme'
+# logo file etc should be in html_static_path, e.g. _static
+html_theme_options = {
+#    'custom_css':'bettercolors.css',
```

Comment:
I don't see bettercolors.css in this pr.

Reply:
And I'd rather call the CSS file site.css in all sites, so that it's just the contents of the file that differs, not the name.  The name can never capture anything of value about the contents, anyway; it's always just the CSS for this site.

Reply:
Agreed. I think this has already been raised as an nbsite issue.